package.json

```diff
@@ -6,7 +6,7 @@
     "backbone.paginator": "~2.0.3",
     "backbone-validation": "~0.11.5",
     "coffee-script": "1.6.1",
-    "edx-pattern-library": "git://github.com/Stanford-Online/ux-pattern-library.git#master",
+    "edx-pattern-library": "git://github.com/Stanford-Online/ux-pattern-library.git#v0.18.1-stanford",
```

Comment:
Hopefully it doesn't matter, but technically this, `0.18.1-stanford` is lower than `0.18.1` since the hyphenated part is a _pre_release qualifier :\

Reply:
true, let me give that a try locally

Reply:
it seems to work?